Makefile

```diff
@@ -98,6 +98,8 @@ ifeq ($(GOBIN),)
 GOBIN := $(FIRST_GOPATH)/bin
 endif
 
+export PATH := $(PATH):$(GOPATH)
```

Comment:
I think this should be GOBIN and not GOPATH

Reply:
Good catch, thanks!